src/absil/ilread.fs

```diff
@@ -1029,377 +1032,373 @@ let count c =
 #endif
         
 
-let seekReadUInt16Adv ctxt (addr: byref<int>) =  
```

Comment:
These changes are a problem - the use of ``byref`` was introduced to eliminate a of tuple allocations.  Using a struct tuple or struct return would be acceptable I think (though I'd kind of prefer if we could freely use byref args in the comppiler codebase, where perf can be critical....)